docs/book/src/topics/using-iam-roles-in-mgmt-cluster.md

```diff
@@ -0,0 +1,48 @@
+# Using IAM roles in management cluster instead of AWS credentials
+
+## Overview
+
+Sometimes users might want to use IAM roles to deploy management clusters. If the user already has a management cluster which was created using the AWS credentials, CAPA provides a way to use IAM roles instead of using these credentials.
+
+## Pre-requisites
+User has a bootstrap cluster created with AWS credentials. These credentials can be temporary as well.
+To create temporary credentials, please follow [this doc](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_request.html).
+
+We can verify whether this bootstrap cluster is using AWS credentials by checking the `capa-manager-bootstrap-credentials` secret created in `capa-system` namespace:
+```bash
+kubectl get secret -n capa-system capa-manager-bootstrap-credentials -o=jsonpath='{.data.credentials}' | { base64 -d 2>/dev/null || base64 -D; }
+```
+which will give output similar to below:
+```bash
+[default]
+aws_access_key_id = <your-access-key>
+aws_secret_access_key = <your-secret-access-key>
+region = us-east-1
+
+aws_session_token = <session-token>
+```
+
+## Goal
+Create a management cluster which uses instance profiles (IAM roles) attached to EC2 instance.
+
+## Steps for CAPA-managed clusters
+1. Create a workload cluster on existing bootstrap cluster. Refer [quick start guide](https://cluster-api.sigs.k8s.io/user/quick-start.html) for more details.
+  Control plane nodes on the workload cluster will have IAM roles attached which is enough for the target management cluster to work. We want to move the bootstrap cluster to this workload cluster to turn it to a management cluster.
+
+> **Note:** A cluster with a single control plane node won’t be sufficient here due to the `NoSchedule` taint.
```

Comment:
This does not seem correct, we specifically want to use control plane nodes as they have the proper IAM permissions.

I added details on how to move the management cluster, feel free to add some details of that sort here:

```
Since only control-plane nodes have the required IAM roles attached, CAPA deployment should have the necessary tolerations for master (control-plane) node and node selector for master. To modify the CAPA deployment before running clusterctl init, check [cluster api book](https://cluster-api.sigs.k8s.io/clusterctl/commands/init.html).
Run clusterctl init --infrastructure aws on the workload cluster by setting export AWS_B64ENCODED_CREDENTIALS="Cg==" (equivalent to empty string)
```

Reply:
I have mentioned about zeroing the credentials in point 4 below, just before running clusterctl init. This line makes more sense 

> Since only control-plane nodes have the required IAM roles attached, CAPA deployment should have the necessary tolerations for master (control-plane) node and node selector for master.

as compared to

> Control plane nodes on the workload cluster will have IAM roles attached which is enough for the target management cluster to work. We want to move the bootstrap cluster to this workload cluster to turn it to a management cluster.

 Is it ok if I change this line and then keep rest of the instructions as is?